contracts/controller/MarketEnabled.sol

```diff
@@ -76,10 +77,11 @@ contract MarketEnabled is NutzEnabled {
     return amountBabz;
   }
 
-  function sell(address _from, uint256 _amountBabz) public onlyNutz whenNotPaused {
+  function sell(address _from, uint256 _price, uint256 _amountBabz) public onlyNutz whenNotPaused {
     uint256 effectiveFloor = floor();
     uint256 INFINITY = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
     require(effectiveFloor != INFINITY);
+    require(_price == effectiveFloor);
```

Comment:
interesting, i was thinking that the tx would be simulated before, and then not executed, if it fails.
but you are right, due to tx ordering in block, which is determined by miners, it could still happens that simulation is successful, actual tx fails. 🤔